static/css/cf-enhancements.less

```diff
@@ -500,3 +500,18 @@ tbody {
     name: EOF
     eof: true
 */
+
+/**
+ * This is a temporary hack to fix the contact expandables. CF Expandables could use a
+ * larger rework, especially removing duplicated blocks (open/close states, double search
+ * filter buttons based on device size, etc)
+ */
+.expandable__padded .expandable_header {
+  padding-right: 90px;
```

Comment:
It's not that generous, it's only 20px 

![screen shot 2015-03-12 at 5 09 18 pm](https://cloud.githubusercontent.com/assets/1280430/6628311/be7b7332-c8da-11e4-9d20-d32974282909.png)

Worth noting that the 90px includes the 16px that already existed.